martus-meta/source/org/martus/meta/TestSha1ToSha2Change.java

```diff
@@ -132,6 +149,94 @@ public void testSha1ClientUploadBulletinToSha2Server() throws Exception
 		TRACE_END();
 	}
 
+	public void testSha1ClientShareBulletinWithSha1Contact() throws Exception
+	{
+		TRACE_BEGIN("testSha1ClientShareBulletinWithSha1Contact");
+
+		addHqForClient(appSha1, hqAppSha1.getAccountId());
+
+		Bulletin b1 = createTestBulletin(appSha1);
+		Bulletin b2 = createTestBulletin(appSha1);
+
+		assertEquals("failed upload1?", NetworkInterfaceConstants.OK, sha1AppUploader.uploadBulletin(b1));
+		assertEquals("failed upload2?", NetworkInterfaceConstants.OK, sha1AppUploader.uploadBulletin(b2));
+
```

Comment:
The two asserts here and other unit tests can be extracted out into a verify(uploader, bulletin, string expectedResult, string actualResult)